### PR TITLE
APEXMALHAR-2161: Add tests for AbstractThroughputFileInputOperator

### PR DESCRIPTION
**Pull request by DT-Priyanka**
Added unit tests for AbstractThroughputFileInputOperator class

### CODE REVIEW

**Comment by DT-Priyanka** [comment]
@ashwinchandrap  please review.

**Comment by DT-Priyanka** [comment]
@yogidevendra  can you please review and merge?

File: library/src/test/java/com/datatorrent/lib/io/fs/AbstractThroughputFileInputOperatorTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    mockOperatorStats.counters = fileCountersMock;
+    when(mockPartition.getStats()).thenReturn(mockBatchStats);
+    when(mockBatchStats.getLastWindowedStats()).thenReturn(Collections.singletonList(mockOperatorStats));
+    when(fileCountersMock.getCounter(any(FileCounters.class))).thenReturn(fileCounterMock);
+    when(fileCounterMock.getValue()).thenReturn(20L);
+
+    Response response = underTest.processStats(mockBatchStats);
+
+    Assert.assertTrue(response.repartitionRequired);
```
Comment (yogidevendra) [inline library/src/test/java/com/datatorrent/lib/io/fs/AbstractThroughputFileInputOperatorTest.java:99]: Could you please add test scenario for repartitionRequired=false.

File: library/src/test/java/com/datatorrent/lib/io/fs/AbstractThroughputFileInputOperatorTest.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  }
+
+  @Test
+  public void testRepartitioning()
+  {
+    underTest.setPartitionCount(4);
```
Comment (yogidevendra) [inline library/src/test/java/com/datatorrent/lib/io/fs/AbstractThroughputFileInputOperatorTest.java:128]: Please add more scenarios for repartitioning.

**Pull request merged**